k9mail/src/main/java/com/fsck/k9/message/PgpMessageBuilder.java

```diff
@@ -182,7 +182,7 @@ private Intent buildOpenPgpApiIntent(boolean shouldSign, boolean shouldEncrypt,
 //                pgpApiIntent.putExtra(OpenPgpApi.EXTRA_ENCRYPT_OPPORTUNISTIC, cryptoStatus.isEncryptionOpportunistic());
             }
         } else {
-            pgpApiIntent = new Intent(isPgpInlineMode ? OpenPgpApi.ACTION_SIGN : OpenPgpApi.ACTION_DETACHED_SIGN);
+            pgpApiIntent = new Intent(isPgpInlineMode ? OpenPgpApi.ACTION_CLEARTEXT_SIGN : OpenPgpApi.ACTION_DETACHED_SIGN);
```

Comment:
@Valodim will have to look at this I think (and the commented out code visible in the line above.